.stylelintrc.json

```diff
@@ -1,7 +1,6 @@
 {
     "extends": [
-      "stylelint-config-sass-guidelines",
-      "stylelint-config-prettier"
```

Comment:
why did you delete it?

Reply:
My oversight, restoring to the previous state.